modules/combined/tests/fieldsplit_contact/2blocks3d.i

```diff
@@ -9,7 +9,7 @@
 []
 
 [Problem]
-  error_on_jacobian_nonzero_reallocation = true;
+  error_on_jacobian_nonzero_reallocation = true
 []
```

Comment:
I will add a better example.  The contacted surfaces do not change for this case.